module/merintr/alias.c

```diff
@@ -251,12 +252,25 @@ void AliasSave(void)
       // Save function key aliases
       sprintf(temp, "F%d", i + 1);
 
+      len = (int)strlen(aliases[i].text);
       if (aliases[i].cr)
-	 strcpy(text, aliases[i].text);
+      {
+	      strcpy(text, aliases[i].text);
+         command = A_TEXTCOMMAND;
+      }
       else
-	 sprintf(text, "%s~", aliases[i].text);
-
+      {
+         sprintf(text, "%s~", aliases[i].text);
+         command = A_TEXTINSERT;
+      }
       WritePrivateProfileString(fullSection, temp, text, cinfo->ini_file);
+
+      if (gbClassicKeys)
```

Comment:
This exactly duplicates existing code.  Make a shared function instead.

Reply:
Created new function `UpdateKeyTables` as suggested and called in within `AliasInit` and `AliasSave`